packages/intrepid2/src/Discretization/Basis/Intrepid2_HierarchicalBasisFamily.hpp

```diff
@@ -57,9 +57,18 @@
 #include "Intrepid2_LegendreBasis_HVOL_LINE.hpp"
 
 namespace Intrepid2 {
-  // the following defines a family of hierarchical basis functions that matches the unpermuted ESEAS basis functions
-  // each basis member is associated with appropriate subcell topologies, making this suitable for continuous Galerkin finite elements.
   
+
+//Dummy basis to be temporarily used for Hierarchical bases that have not been implemented yet
+  template<typename ExecutionSpace, typename OutputScalar, typename PointScalar>
+  class dummyBasis
+  : public Basis<ExecutionSpace,OutputScalar,PointScalar> {
+  public:
+    dummyBasis(int /*order*/) {};
+  };
+
```

Comment:
I like the dummyBasis as a way to allow defining the full Lagrangian family in advance of having the full hierarchical family implemented.